app/domain/authentication/authn_jwt/signing_key/fetch_jwks_uri_signing_key.rb

```diff
@@ -17,12 +17,12 @@ def initialize(
           logger: Rails.logger
         )
           @logger = logger
-
-          @fetch_required_secrets = fetch_required_secrets
-          @resource_class = resource_class
-          @resource_id = authentication_parameters.authn_jwt_variable_id_prefix
           @http_lib = http_lib
           @create_jwks_from_http_response = create_jwks_from_http_response
+          @fetch_required_secrets = fetch_required_secrets
+          @resource_class = resource_class
+
+          @variable_id = authentication_parameters.authn_jwt_variable_id_prefix
```

Comment:
i think we should ditch the `@variable_id` param in all the PR and use `authentication_parameters.authn_jwt_variable_id_prefix` directly

Reply:
Done